starter/schemaorg/Makefile

```diff
@@ -93,77 +91,80 @@ custom_train_nlu_flags ?=
 	mv $@.tmp $@
 
-$(experiment)/synthetic-d%.tsv: $(experiment)/schema.tt $(dataset_file) $(geniedir)/languages/thingtalk/en/*.genie
+multidomain/manifest.tt: $(addsuffix /manifest.tt,$(all_experiments))
+	cat $^ > $@
+
+%/synthetic.tsv: %/manifest.tt $(dataset_file) $(geniedir)/languages/thingtalk/en/*.genie
 	$(genie) generate \
 	  --template $(geniedir)/languages/$(template_file) \
-	  --thingpedia $(experiment)/schema.tt --entities $(experiment)/entities.json --dataset $(dataset_file) \
+	  --thingpedia $*/manifest.tt --entities $*/entities.json --dataset $(dataset_file) \
 	  --target-pruning-size $(target_pruning_size) \
-	  -o $@.tmp --no-debug $(generate_flags) --maxdepth $$(echo $* | cut -f1 -d'-') \
+	  -o $@.tmp --no-debug $(generate_flags) --maxdepth $(maxdepth) \
 	  --random-seed $@ --id-prefix $*:
 	mv $@.tmp $@
 
-$(experiment)/synthetic.tsv : $(foreach v,1 2 3,$(experiment)/synthetic-d6-$(v).tsv) $(experiment)/synthetic-d8.tsv
-	cat $^ > $@
-
 shared-parameter-datasets.tsv:
 	$(thingpedia_cli) --url $(thingpedia_url) --developer-key $(developer_key) --access-token invalid \
 	  download-entity-values --manifest $@ --append-manifest -d shared-parameter-datasets
 	$(thingpedia_cli) --url $(thingpedia_url) --developer-key $(developer_key) --access-token invalid \
 	  download-string-values --manifest $@ --append-manifest -d shared-parameter-datasets
 
-$(experiment)/parameter-datasets.tsv : $(experiment)/schema.trimmed.tt $(experiment)/data.json shared-parameter-datasets.tsv
+%/parameter-datasets.tsv : %/schema.trimmed.tt %/data.json shared-parameter-datasets.tsv
 	sed 's|\tshared-parameter-datasets|\t../shared-parameter-datasets|g' shared-parameter-datasets.tsv > $@
-	$(genie) make-string-datasets --manifest $@.local -d $(experiment)/parameter-datasets --thingpedia $(experiment)/schema.trimmed.tt --data $(experiment)/data.json --class-name $($(experiment)_class_name) --dataset schemaorg
+	$(genie) make-string-datasets --manifest $@.local -d $*/parameter-datasets --thingpedia $*/schema.trimmed.tt --data $*/data.json --class-name $($(*)_class_name) --dataset schemaorg
 	cat $@.local >> $@
 	rm $@.local
 
-$(experiment)/augmented.tsv : $($(experiment)_paraphrase) $(experiment)/synthetic.tsv $(experiment)/parameter-datasets.tsv
-	$(genie) augment -o $@.tmp -l en-US --thingpedia $(experiment)/schema.tt --parameter-datasets $(experiment)/parameter-datasets.tsv \
+%/augmented.tsv : %/synthetic.tsv %/parameter-datasets.tsv
+	$(genie) augment -o $@.tmp -l en-US --thingpedia $*/manifest.tt --parameter-datasets $*/parameter-datasets.tsv \
 	  --synthetic-expand-factor 1 --quoted-paraphrasing-expand-factor 60 --no-quote-paraphrasing-expand-factor 20 --quoted-fraction 0.0 \
-	  --debug $($(experiment)_paraphrase) $(experiment)/synthetic.tsv
+	  --debug $($(*)_paraphrase) $*/synthetic.tsv
 	mv $@.tmp $@
 
+multidomain/augmented.tsv: $(addsuffix /augmented.tsv,$(all_experiments))
+	cat $^ > $@
```

Comment:
Are we missing the rule to make `multidomain/eval/annotated.tsv` from the other files?